src/object/object_manager.h

```diff
@@ -303,6 +303,7 @@ class CObjectManager : public CSingleton<CObjectManager>
     //@}
 
 private:
+    float ClampPower(ObjectType type, float power); //!Prevents creation of overcharged power cells
```

Comment:
This comment applies to CleanRemovedObjectsIfNeeded, not ClampPower. Either move it to the line above, or replace `//!` with `//<` (the first variant would be preferred in most cases). There should also be a space between `!` and the text.

See https://www.doxygen.nl/manual/docblocks.html